recipe/build.sh

```diff
@@ -2,6 +2,13 @@
 
 export LDFLAGS="-L${PREFIX}/lib"
 export CFLAGS="${CFLAGS} -O3 -I${PREFIX}/include"
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
+    brew uninstall gmp
```

Comment:
Possibly, if `conda` sets it somehow. Though `conda` makes sure nothing is leaking in from the users environment. Namely all environment variables are unset.